modules/javafx.graphics/src/main/native-glass/mac/GlassApplication.m

```diff
@@ -1050,6 +1067,18 @@ + (BOOL)syncRenderingDisabled {
     return !isEmbedded;
 }
 
+/*
+ * Class:     com_sun_glass_ui_mac_MacApplication
+ * Method:    _isNormalTaskbarApp
+ * Signature: ()Z;
+ */
+JNIEXPORT jboolean JNICALL Java_com_sun_glass_ui_mac_MacApplication__1isNormalTaskbarApp
+(JNIEnv *env, jobject japplication)
+{
+    LOG("Java_com_sun_glass_ui_mac_MacApplication__1isNormalTaskbarApp");
+    return isNormalTaskbarApp;
```

Comment:
Are we guaranteed that this is not called before the runLoop is invoked? The `isNormalTaskbarApp` is only set to `YES` at line 536, and in theory, this method may get called before.

Reply:
Yes, we only call it after runLoop (which is why the check is in the `wrappedRunnable`). A comment seems in order, so I'll add one.